docs/Set-PSReadLineOption.md

```diff
@@ -1,256 +1,274 @@
 ---
+external help file: Microsoft.PowerShell.PSReadLine.dll-Help.xml
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: PSReadLine
+ms.date: 12/07/2018
+online version: http://go.microsoft.com/fwlink/?LinkId=821453
 schema: 2.0.0
-external help file: Microsoft.PowerShell.PSReadLine2.dll-help.xml
+title: Set-PSReadLineOption
 ---
 
 # Set-PSReadLineOption
 
 ## SYNOPSIS
-
 Customizes the behavior of command line editing in PSReadLine.
 
 ## SYNTAX
 
+### OptionsSet
+
 ```
 Set-PSReadLineOption
  [-EditMode <EditMode>]
- [-HistorySavePath <String>]
- [-HistorySaveStyle <HistorySaveStyle>]
+ [-ContinuationPrompt <String>]
  [-HistoryNoDuplicates]
- [-HistorySearchCaseSensitive]
- [-PromptText <string>]
- [-ExtraPromptLineCount <Int32>]
- [-Colors <Hashtable>]
  [-AddToHistoryHandler <Func[String, Boolean]>]
  [-CommandValidationHandler <Action[CommandAst]>]
- [-ContinuationPrompt <String>]
  [-HistorySearchCursorMovesToEnd]
  [-MaximumHistoryCount <Int32>]
  [-MaximumKillRingCount <Int32>]
  [-ShowToolTips]
+ [-ExtraPromptLineCount <Int32>]
  [-DingTone <Int32>]
  [-DingDuration <Int32>]
  [-BellStyle <BellStyle>]
  [-CompletionQueryItems <Int32>]
- [-WordDelimiters <string>]
- [-AnsiEscapeTimeout <int>]
+ [-WordDelimiters <String>]
+ [-HistorySearchCaseSensitive]
+ [-HistorySaveStyle <HistorySaveStyle>]
+ [-HistorySavePath <String>]
+ [-AnsiEscapeTimeout <Int32>]
+ [-PromptText <String>]
  [-ViModeIndicator <ViModeStyle>]
- [-ViModeChangeHandler <ScriptBlock>]
+ [-Colors <Hashtable>]
+ [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-The Set-PSReadLineOption cmdlet is used to customize the behavior of the PSReadLine module when editing the command line.
+The `Set-PSReadLineOption` cmdlet customizes the behavior of the PSReadLine module when you are editing the command line.
 
 ## EXAMPLES
 
-### --------------  Example 1  --------------
+### Example 1: Set color values for multiple types
 
-This example sets some different colors.
+This example shows three different methods for how to set the color of tokens displayed in PSReadLine.
 
-```
-PS C:\> Set-PSReadLineOption -Colors @{
-    # Use a ConsoleColor enum
-    "Error" = [ConsoleColor]::DarkRed
+```powershell
+Set-PSReadLineOption -Colors @{
+ # Use a ConsoleColor enum
+ "Error" = [ConsoleColor]::DarkRed
 
-    # 24 bit color escape sequence
-    "String" = "$([char]0x1b)[38;5;100m"
+ # 24 bit color escape sequence
+ "String" = "$([char]0x1b)[38;5;100m"
 
-    # RGB value
-    "Command" = "#8181f7"
+ # RGB value
+ "Command" = "#8181f7"
 }
 ```
 
-### --------------  Example 2  --------------
+### Example 2: Set bell style
```

Comment:
This example is probably not a good example because it might not work on some platforms.